doc/performance-investigations/0021-upgrade-python-to-3.10.8/outputs/baseline/2023-01-03/Results_For_Latest_Benchmark.yaml

```diff
@@ -0,0 +1,14 @@
+---
+Percentile Averages:
+50th: 68ms
+90th: 91ms
+95th: 107ms
+99th: 681ms
+99.9th: 1644ms
+---
+GETs (99th): 638ms
+POSTs (99th): 753ms
+---
+Total Requests: 65,122
+Total Failures: 40
```

Comment:
In future prob worth running the baseline first and the test not too long after to ensure the test is consistent. Prob need to investigate these errors to see if they are related to the upgrade and/or maybe running the test again? I don't think I had any errors when I ran my benchmark 🤔

Reply:
Yeah seemed odd, didn't get these errors the first time I ran baseline tests. I'll re run the benchmarks for both baseline and investigation